packages/odie-client/src/components/message/messages-container.tsx

```diff
@@ -219,10 +216,9 @@ export const MessagesContainer = ( { currentUser }: ChatMessagesProps ) => {
 						{ chat.provider === 'odie' && (
 							<>
 								{ ! forceEmailSupport && <ViewMostRecentOpenConversationNotice /> }
-								{ availableStatusWithFeedback.includes( chat.status ) && (
+								{ chat.status === 'sending' && (
 									<div className="odie-chatbox__action-message">
-										{ chat.status === 'sending' && <ThinkingPlaceholder /> }
-										{ chat.status === 'dislike' && <DislikeFeedbackMessage /> }
```

Comment:
We cannot reach this code anymore, so I'm removing this and the DislikeFeedbackMessage component.